wwwbase/admin/bulkReplace.php

```diff
@@ -113,12 +120,21 @@
                    $objCount,
                    $replace);
     FlashMessage::add($msg, 'success');
-    Util::redirect('index.php');
+    if (!empty($objStructured)) {
+      // eventually we could pass as encoded string, but this array may be huge exceeding the limit of URL size
+      //$serialized = rawurlencode(serialize($objStructured)); 
+      $serialized = implode(',', $objStructured); 
+      Session::unsetVar('objStructured');
+      Util::redirect('bulkReplaceStructured.php?objStructured='.$serialized); // case history of changed structured definitions
```

Comment:
Crezi că le putem trimite prin sesiune? Tehnic, ar putea fi mii de ID-uri, fiecare de 6-7 caractere + separatorul. E ușor de depășit limita (neoficială) de 2.000 de caractere pentru lungimea URL-ului. În plus, mi se pare că în sesiune poți stoca direct vectorul (se descurcă el cu serializarea), deci scăpăm de implode-uri și explode-uri.

Reply:
Ceva e dubios! Am căutat puțin [php set cookie and redirect] și toată lumea zice că merge.

Ar putea fi cauzat de `Session::unsetVar()`. Te-ai uitat la comentariul din ea? Dacă ștergi ultima variabilă din sesiune, funcția aia omoară și sesiunea. Dacă după aceea setezi o nouă variabilă (inclusiv un cookie), de fapt în spate nu se mai deschide o nouă sesiune și pierzi ce ai setat.

Dacă nu-i asta cauza, o putem lăsa așa, altfel pare că ne prindem degetele rău de tot.

Reply:
Rămâne totuși problema că trebuie cândva să facem 
```php 
Session::unsetVar('objStructured');
```
după preluarea în `bulkReplaceStructured.php`? Ziceam mai sus... link->back->boom!
Facem un buton de `backToIndex` în șablonul de structurate să se ocupe de variabilă?

Reply:
Mda, veșnica problemă cu back button. Nu m-aș preocupa prea mult... La o adică, din orice pagină POST dacă dai back și forward poți avea probleme, degeaba o rezolvăm în cazul ăsta.

Cel mult poți pune undeva în Javascript un `confirm('Dacă navigați înapoi, veți pierde istoria')` sau pe-acolo.

Reply:
Putem face șablonul `entry.tpl` ca legăturile să permită target="_blank"?